cmd/kubelet/app/server.go

```diff
@@ -202,11 +204,25 @@ is checked every 20 seconds (also configurable with a flag).`,
 			}
 
 			// load kubelet config file, if provided
-			if configFile := kubeletFlags.KubeletConfigFile; len(configFile) > 0 {
+			configFile := kubeletFlags.KubeletConfigFile
```

Comment:
nit: since we're not pre-calculating the len(configFile) or len(configDropinDir) I'm not sure the value in making a new variable. the eventual line
`if len(kubeletFlags.KubeletConfigFile) > 0 || len(kubeletFlags.KubeletDropinConfigDirectory > 0 {` is not horribly long

Reply:
fixed